public/js/components/Sources.js

```diff
@@ -62,9 +62,9 @@ let File = (props) => {
     )
   );
 };
-File = React.createFactory(File);
```

Comment:
linter was complaining that `File` is a builtin type that we are overriding so I had to change these. Will probably pull them out later today too